python/tank/util/qt_importer.py

```diff
@@ -111,7 +125,7 @@ def _import_pyside(self):
 
         :returns: The (binding name, binding version, modules) tuple.
         """
-        from PySide import QtCore, QtGui
+        from PySide import QtCore, QtGui, QtWebKit, QtNetwork
```

Comment:
Are you certain this might not crash on certain DCCs? We might have to `try/except` those two includes.